types/stripe/index.d.ts

```diff
@@ -7667,7 +7667,12 @@ declare namespace Stripe {
             status: Statuses;
 
             /**
-             * Can be card, bank_account, or stripe_account.
+             * A string that identifies this transaction as part of a group.
+             */
+            transfer_group: string;
```

Comment:
Is this always set or can it be `null`?

Reply:
@pushplay Good catch! You are right and I've updated the PR.